aesm-client/src/imp/windows.rs

```diff
@@ -131,9 +132,9 @@ impl AesmClient {
             assert_eq!(spid.len(), 16);
```

Comment:
Will need an assert here to check the nonce length, but let's wait to resolve the other comment first.